docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -10,245 +10,242 @@
 title: LoopBack 4
 url: index.html
 children:
+  - title: 'Getting started'
```

Comment:
I don't know why you change the indentation here. The current formatting should be fine:

```yaml
title: LoopBack 4
url: index.html
children:
- title: 'Getting started' (No extra spaces before -)
```

Reply:
I just checked it and it is automatically done by VS code by the following plugin.

 Prettier - Code formatter 1.6.1

I have disabled it now to keep the current formatting as suggested.

Reply:
I noticed there are some directives for prettier on the file .prettierignore and currently it has an entry for .json files but not for .yml files .  Shall I add and entry for yml files in this file and keep my plugin enabled ? .  I tested and it works by just adding this line on this file.